lib/browser.ts

```diff
@@ -61,6 +62,11 @@ export class Webdriver {
        opt_message?: string) => wdpromise.Promise<any>;
 }
 
+export class ExtendedWebdriver {
```

Comment:
1) For consistency, I'll rename `Webdriver` to `AbstractWebDriver` and this to `AbstractExtendedWebDriver`
2) Yes